static/css/components/header-bar.less

```
@@ -91,13 +91,13 @@
   }
 
   .app-drawer {
-    @topHeight: 50px;
+    @topHeight: 0px;
     position: fixed;
     width: 300px;
     max-width: 100vw;
     right: 0;
     top: @topHeight;
```

```suggestion
    position: fixed;
    width: 300px;
    max-width: 100vw;
    right: 0;
    top: 0;
```
Lets get rid of that variable we don't need anymore.
There seems to be a convention in our the codebase of using 0 instead of 0px so lets stick with that.